tools/topology/topology2/cavs-sdw.conf

```diff
@@ -135,6 +137,7 @@ Object.Pipeline {
 		index 5
 
 		 Object.Widget.copier.1 {
+			dai_index	2
```

Comment:
i find it hard to review this. 

dai_index = 2 is seen on both line 123 and 140. is this a bug or a feature?

Reply:
its a feature. This one is FE DAI index 2 and the other one is BE DAI index 2